### ISSUE TITLE
WORKING FOR REALME NARZO 50 PRO

### ISSUE THREAD

**Issue by killshotxd** [turistu/rmx3474-rooting#12]
Just did that and got success after 1 min applylkunlock checkapprovl it came success and I checked on deeptest its giving me star deep testing 😃.
![Screenshot_2023-10-01-13-04-53-72_5461cd6783b6e8d9f5be0980ff5227f9](https://github.com/turistu/rmx3474-rooting/assets/89957432/eefe95cc-7f5c-446c-ab6c-a1db436ca83d)

**Comment by killshotxd** [comment]
Got this working but phone is not holding on fastboot screen after clicking on start deep testing tried on A0.4 it just reboots to system again 😢

**Comment by turistu** [comment]
> Got this working but phone is not holding on fastboot screen after clicking on start deep testing tried on A0.4 it just reboots to system again 😢

Yes, the same issue as [here](https://github.com/turistu/rmx3474-rooting/issues/6#issuecomment-1662244292), unfortunately.
